app/controllers/sessions_controller.rb

```diff
@@ -2,13 +2,13 @@ class SessionsController < ApplicationController
   def index
     @user_session_ids = current_user.sessions.pluck(:id)
     @sessions = SessionQuery.new(
-      relation: sessions.joins(:location).distinct,
+      relation: sessions&.joins(:location)&.distinct,
       params: filter_params
-    ).call.includes(:attendees, :tags).order(:starts_at)
+    )&.call&.includes(:attendees, :tags)&.order(:starts_at)
   end
 
   def show
-    @session = sessions.friendly.find(params[:id])
+    @session = sessions&.friendly&.find(params[:id])
   end
 
   private
```

Comment:
I would opt to change the sessions so we dont need to add too many safe operators.
This would allow us to remove all safe operators from here and form the session_query.rb
```suggestion
  private
  
    def sessions
       Session.where(conference: current_conference)
       # OR
       current_conference&.sessions || Session.none
  end
```